src/common/Common.ts

```diff
@@ -78,40 +78,49 @@ export class CommandLineUtils {
 
 // tslint:disable-next-line: max-classes-per-file
 export class Version {
+    public static from(input: string): Version {
+        // support version strings using - or . as separators (e.g 13-0-4 and 13.0.4)
+        const parts = input.trim().toLowerCase().replace(/-/gi, '.').split('.');
+        const major = parts.length >= 1 ? Number.parseInt(parts[0], 10) : 0;
```

Comment:
This is okay, I guess. It's very loose/permissive with respect to its input. For example, it's going to be getting input strings like `27;google_apis;x86`, and translating those into `27.0.0`. That's certainly a reach from standard versioning conventions, but it will work for our purposes.

Reply:
will fix